src/Tizen.NET.MaterialComponents/Components/MItem.cs

```diff
@@ -0,0 +1,26 @@
+using ElmSharp;
+
+namespace Tizen.NET.MaterialComponents
+{
+    public class MItem
+    {
+        public MListStyle Style { get; set; }
+        public string Icon { get; set; }
+
+        public string Title { get; set; }
+
+        public string SubText { get; set; }
+
+        public string MetaText { get; set; }
+
+        public EvasObject Control { get; set; }
+
+        public MItem(string title, string icon = null)
```

Comment:
avoid optional parameter as a public API. Is is necessary>

Reply:
Done